### ISSUE TITLE
No exported member 'PerfectScollbarModule'

### ISSUE THREAD

**Issue by hbweb** [zefoy/ngx-perfect-scrollbar#58]
Hi guys,
Thanks for writing this wrapper :) Pretty good stuff. 
Just a quick head up when using it. I noticed there is a typescript error when import the PerfectScollbarModule in App.module.ts 

` import {PerfectScollbarModule} from 'ngx-perfect-scrollbar' `

It gave a warning as below:

```
[ts] Module './node_modules/ngx-perfect-scrollbar/dist/index' has no exported member 'PerfectScollbarModule'
```

Thanks

**Comment by hbweb** [comment]
aah It looks like it has been resolved in version 4.5.0 ?

**Comment by sconix** [comment]
That sounds like a problem in your build configuration. At least I haven't seen this error and usually that is sign of something wrong with the build setup.
